README.md

```diff
@@ -47,7 +47,14 @@ apt-get install xrdp
 
 ### RedHat / CentOS / Fedora
 ```bash
-yum install xrdp
+yum -y install xrdp
+dnf -y install xrdp
```

Comment:
I know what `-y` does but why it's added?
I think users should confirm it on their own responsibility.

`apt` also ask users to confirm installation. Adding it only to `yum/dnf` is inconsistent.